image_store.go

```diff
@@ -109,8 +109,9 @@ func (s *remoteImages) Delete(ctx context.Context, name string, opts ...images.D
 		}
 	}
 	_, err := s.client.Delete(ctx, &imagesapi.DeleteImageRequest{
-		Name: name,
-		Sync: do.Synchronous,
+		Name:   name,
+		Sync:   do.Synchronous,
+		Target: descToProto(do.Target),
```

Comment:
nit: maybe we can reuse these helpers added in https://github.com/containerd/containerd/pull/8757

https://github.com/containerd/containerd/blob/fb786b2f17b2c7d7ea5d9c9009d980bc03379952/oci/spec.go#L247

https://github.com/containerd/containerd/blob/fb786b2f17b2c7d7ea5d9c9009d980bc03379952/oci/spec.go#L258